README.md

```diff
@@ -62,6 +62,11 @@ Options: [defaults]
                                         [no]
   --icon-muted <icon>                   icon to use when muted [none]
   --icon-sink <icon>                    icon to use for sink [none]
+  --format <string>                     use a format string to control the output
+                                        [$VOL_ICON ${VOL_LEVEL}%  $ICON_SINK $SINK_NICKNAME]
```

Comment:
also, sorry for missing this earlier: the default value should be at the end of the message